### PR TITLE
docs: add v8 migration guide

### LINKED ISSUE #1701: Docs: add migration guide for version 8 release
Documents

- [ ] CLI Update instructions
- [ ] New Features/Breaking Changes
- [ ] Provided Migrations
- [ ] Latest dependencies

### PR DESCRIPTION
**Pull request by timdeschryver**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #1701

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by timdeschryver** [comment]
Is there any breaking change that I have missed that you know of?
I'll take a look at the commits in a couple of days, to double check.

File: projects/ngrx.io/content/guide/migration/v8.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+## Angular CLI update
+
+NgRx supports using the Angular CLI `ng update` command to update your dependencies.
+
+To update your packages to the latest released version, run the command below.
+
+```sh
+ng update @ngrx/store
+```
```
Comment (brandonroberts) [inline projects/ngrx.io/content/guide/migration/v8.md]: We might need a section about the migrations that are run when you update.

**Comment by brandonroberts** [comment]
We can follow up with the migrations if necessary. I wanted this to be in the first beta

File: projects/ngrx.io/content/guide/migration/v8.md (hunk 1 above)
Comment (timdeschryver) [reply projects/ngrx.io/content/guide/migration/v8.md]: Yep, I'll verify them when we've published the beta version.

**Pull request merged**